### ISSUE TITLE
Power Plug Firmware 5.03 Unstable

### ISSUE THREAD

**Issue by bugale** [TomerFi/aioswitcher#691]
### What happened?

TL;DR:
My power plug with firmware 5.03 can't be controlled through `aioswitcher` after startup/wifi connection.  The official app is able to control is locally. `aioswitcher` succeeds eventually after some time/retries.

After my previous power plug broke, I received a new one, which got a forced update from Switcher's devs to firmware version 5.03, which at least the Android app wasn't aware of.
Since I didn't have it for too long before the upgrade, it is hard to say if the issue is due to the firmware version, or the specific instance I have.
I am also not quite sure what causes the issue to reproduce, or to get solved, but it looks like the issue reproduces sometimes after the power plug resets, or the router resets, and gets "fixed" by itself only after I play with it a bit and try controlling the device using different methods.

I have captured a PCAP file from my router during one of the times the issue reproduced, and it is attached [here](https://github.com/TomerFi/aioswitcher/files/12014368/test.zip).
In the PCAP there is a total of 4 IPs:
`47.91.75.117` - Switcher's server? (I assume)
`192.168.1.38` - The power plug
`192.168.1.5` - HomeAssistant from which I tried to turn on the device
`192.168.1.2` - My PC from which I ran the following code a few times:
```
from aioswitcher import *
from aioswitcher.api import *
import asyncio

async def control_power_plug(device_ip, device_id, command):
    async with SwitcherType1Api(device_ip, device_id) as api:
        await api.control_device(command)
asyncio.run(control_power_plug("192.168.1.38", "6cfc1f", Command.ON))
```

I claimed above the app succeeds to control the device locally, though I now notice that in this PCAP it didn't happen, and the app decided to use their server, but I did see it happen in the past and lost the evidence :)
I will try to reproduce it and attach a PCAP with these evidence later.

Anyway, in the attached PCAP it can be seen that in the beginning, both the PC and HomeAssistant are able to send a login request and receive a response, but when they send the "turn on" command, the device just closes the connection without sending a response (and doesn't turn on).
After a few tries, even though the exact same data is sent in the TCP streams (except for timestamp/checksum), the device does send a response (and actually turn on).

It is kind of a philosophical question whether this is a bug in `aioswitcher` or the device.
One could argue that since the app is able to control it locally always (again, will provide evidence later), it is a bug in `aioswitcher`.
Even if this ticket will be deemed as a bug in the device rather than `aioswitcher`, I believe it is beneficial to have this ticket even just for reference, or for sending to Switcher's devs.

When I saw the local API working in their app, I did notice it sends slightly different data than `aioswitcher`. In a few places where `aioswitcher` just puts `\x00`s, the app fills in actual data. It might be the reason there is a difference in behavior.

### Module Version

3.3.0

### Device Type

Switcher Power Plug

### Firmware Version

5.03

**Comment by bugale** [comment]
I have reproduced the issue and tried controlling the device locally using the official app, and it seems from this [PCAP](https://github.com/TomerFi/aioswitcher/files/12053600/test.zip) that even though the app writes "Local" in its GUI, and actually tries to send packets locally, it "fails" for some reason and fallsback to controlling through cloud. (`192.168.1.126` is the app)
If so, my previous statement that the app is able to control it locally might have been false.
